### ISSUE TITLE
Klaxon can't parse int array (can't find constructor)

### ISSUE THREAD

**Issue by Litarvan** [cbeust/klaxon#219]
Klaxon drops an exception when parsing an object with an int array from a model, managed to reproduce with a short code : 

```
import com.beust.klaxon.Klaxon

data class Test(val values: Array<Int>) // Tested with default "= emptyArray()" too

fun main(args: Array<String>) {
    val test: Test = Klaxon().parse("""
        {
          "values": [1,2,4]
        }
    """.trimIndent())!!
}
```

```
Exception in thread "main" com.beust.klaxon.KlaxonException: Couldn't find a suitable constructor for class Test to initialize with {values=[I@32eff876}: java.lang.IllegalArgumentException argument type mismatch
	at com.beust.klaxon.JsonObjectConverter.fromJson(JsonObjectConverter.kt:48)
	at com.beust.klaxon.DefaultConverter.fromJsonObject(DefaultConverter.kt:200)
	at com.beust.klaxon.DefaultConverter.fromJson(DefaultConverter.kt:30)
	at com.beust.klaxon.Klaxon.fromJsonObject(Klaxon.kt:247)
	at TestKt.main(test.kt:16)
```

I'm using Klaxon `3.0.1`

**Comment by Litarvan** [comment]
Just found out it works using "IntArray" instead of "Array<Int>", but i think it remains a bug

**Comment by cbeust** [comment]
Definitely a bug, will look into it.

**Comment by rewqazxv** [comment]
Exception becomes
```
Exception in thread "main" com.beust.klaxon.KlaxonException: Unable to instantiate Test with parameters [values: [I@60f00693]
	at com.beust.klaxon.JsonObjectConverter.initIntoUserClass(JsonObjectConverter.kt:94)
	at com.beust.klaxon.JsonObjectConverter.fromJson(JsonObjectConverter.kt:32)
	at com.beust.klaxon.DefaultConverter.fromJsonObject(DefaultConverter.kt:204)
	at com.beust.klaxon.DefaultConverter.fromJson(DefaultConverter.kt:29)
	at com.beust.klaxon.Klaxon.fromJsonObject(Klaxon.kt:294)
	at MainKt.main(main.kt:14)
```
On Klaxon 5.0.1

**Comment by cbeust** [comment]
Should be fixed in 5.0.13.
